package.json

```diff
@@ -84,10 +89,12 @@
           "type": "string",
           "enum": [
             "babylon",
-            "flow"
+            "flow",
+            "typescript",
+            "postcss"
```

Comment:
I've changed it back to flow and babylon and changed the description. I feel this option is super misleading as it should only really be set if there's a blocker bug in one parser that forces you to need another.